PR #21 Reformatted (#128)
mongodb/docs-meta: source/style-guide/markup/format/lists.txt

=====
Lists
=====

Lists are body elements that organize information into adjacent blocks of
markup. Each level of a list must start and end with a blank line.

> Learn more about when to use lists from the :ref:`<lists>` style guide entry.

Reviewer: ```suggestion
To learn more about when to use lists, see the :ref:`<lists>` style guide entry.
```

            - List item 2a
            - List item 2b

Ordered Lists
~~~~~~~~~~~~~

Ordered lists append a number before each item to show a sequential order.

- Use a number followed by a period for the first list item (``1.``,
  ``a.``, and so on). This sets the numbering scheme you are using for this
  level of the list.
- Use a hash character followed by a period (``#.``) for each subsequent
  list item. The parser replaces the hash with the next number in the
  sequence.
- To create a multi-level list, indent each level three spaces for each
  subordinate level. Each subordinate level must start and end with a
  blank line.

.. list-table::
   :header-rows: 1
   :widths: 50 50

   * - Use
     - Avoid

   * -
       .. code-block:: rst

          1. List item 1

             a. List item 1a
             #. List item 1b

          #. List item 2

             a. List item 2a
             #. List item 2b

     -

       .. code-block:: rst
          :copyable: false

          1. List item 1
             a. List item 1a
             b. List item 1b
          2. List item 2
             a. List item 2a
             b. List item 2b

Definition Lists
~~~~~~~~~~~~~~~~

> Definition lists specify a term and a related block that can be used to
> describe the term.

Reviewer: **Style:** See [Use Active Voice](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/).

```suggestion
Definition lists specify a term and a related block that you can use to
describe the term.
```

> Learn more about definition lists from the `{+docutils+} documentation <http://www.docutils.org/docs/ref/rst/restructuredtext.html#definition-lists>`__.

Reviewer: ```suggestion
To learn more about definition lists, see the `{+docutils+} documentation <http://www.docutils.org/docs/ref/rst/restructuredtext.html#definition-lists>`__.
```
